client/src/main/java/org/eclipse/hono/client/impl/AbstractRequestResponseClient.java

```diff
@@ -510,13 +531,31 @@ private R getRequestResponseResult(final Message message) {
      *                  (see {@link AbstractHonoClient#setApplicationProperties(Message, Map)}
      */
     private Message createMessage(final String subject, final Map<String, Object> appProperties) {
+        return createMessage(subject, getDefaultMessageTargetAddress(), appProperties);
+    }
+
+    /**
+     * Build a Proton message with a provided subject (serving as the operation that shall be invoked).
+     * The message can be extended by arbitrary application properties passed in.
+     *
+     * @param subject The subject system property of the message.
+     * @param requestTargetAddress The target address of the message.
```

Comment:
this is just the *address* of the message (the value that goes into the *to* property), I would say.